modules/light-clients/06-solomachine/client_state_test.go

```diff
@@ -21,18 +14,6 @@ const (
 	testPortID                   = "testportid"
 )
 
-func (suite *SoloMachineTestSuite) TestStatus() {
-	clientState := suite.solomachine.ClientState()
-	// solo machine discards arguments
-	status := clientState.Status(suite.chainA.GetContext(), nil, nil)
-	suite.Require().Equal(exported.Active, status)
-
-	// freeze solo machine
-	clientState.IsFrozen = true
-	status = clientState.Status(suite.chainA.GetContext(), nil, nil)
-	suite.Require().Equal(exported.Frozen, status)
-}
-
 func (suite *SoloMachineTestSuite) TestClientStateValidateBasic() {
```

Comment:
maybe this should be `Validate` instead of `ValidateBasic` but super tiny naming nit